### ISSUE TITLE
rsync_from_ncbi.pl: unexpected FTP path (new server?) for 

### ISSUE THREAD

**Issue by MjelleLab** [DerrickWood/kraken2#525]
I have installed using conda, however, I get the following error. 
Tried changing "ftp" to "https" in the rsync_from_ncbi.pl file, but did not help. 

~/software/VIRUSBreakend$ kraken2-build --download-library human --db db
mv: replace 'assembly_summary.txt', overriding mode 0444 (r--r--r--)? y
rsync_from_ncbi.pl: unexpected FTP path (new server?) for https://ftp.ncbi.nlm.nih.gov/genomes/all/GCF/000/001/405/GCF_000001405.39_GRCh38.p13

**Comment by jenniferlu717** [comment]
@MjelleLab we are working on fixing some of the issues with ftp. NCBI recently made some changes to their ftp links which have broken our scripts. Hopefully this will be fixed soon.

**Comment by Lcornet** [comment]
Is it solved?

When i try to create the standad db with a fresh install of kraken, i have this error and i am not sure that it linked to NCBI links:

kraken2-build --standard --db STD --threads 20

Downloading nucleotide gb accession to taxon map... done.
Downloading nucleotide wgs accession to taxon map... done.
Downloaded accession to taxon map(s)
Downloading taxonomy tree data... done.
Uncompressing taxonomy data... done.
Untarring taxonomy tree data... done.
Step 1/2: Performing rsync file transfer of requested files
Rsync file transfer complete.
Step 2/2: Assigning taxonomic IDs to sequences
All files processed, cleaning up extra sequence files... done, library complete.
Masking low-complexity regions of downloaded library... done.
Step 1/2: Performing rsync file transfer of requested files
Rsync file transfer complete.
Step 2/2: Assigning taxonomic IDs to sequences
All files processed, cleaning up extra sequence files... done, library complete.
Masking low-complexity regions of downloaded library... done.
Step 1/2: Performing rsync file transfer of requested files
Rsync file transfer complete.
Step 2/2: Assigning taxonomic IDs to sequences
All files processed, cleaning up extra sequence files... done, library complete.
Masking low-complexity regions of downloaded library... done
Downloading plasmid files from FTP....
awk: cannot open .listing (No such file or directory)

**Comment by aeu79** [comment]
> Is it solved?

Yes, but there is no new release (I just tested it in a fresh install). After installing with install_kraken2.sh, go to the "KRAKEN2_DIR" and modify these two files:

**download_genomic_library.sh** (line 17)
FTP_SERVER="**ftp**://$NCBI_SERVER"
to
FTP_SERVER="**https**://$NCBI_SERVER"

**rsync_from_ncbi.pl** (line 46)
if (! ($full_path =~ s#^ftp://${qm_server}${qm_server_path}/##)) {
to
if (! ($full_path =~ s#^https://${qm_server}${qm_server_path}/##)) {

Source:
[Commit 1](https://github.com/DerrickWood/kraken2/commit/493568d3f44753f8f711634f86fbaab1b9dfb071)
[Commit 2](https://github.com/DerrickWood/kraken2/commit/4f648f5a83498f666193be7b9b71a38e3b99ee24)

**Comment by sbs87** [comment]
There are also pre-build databases here:
https://benlangmead.github.io/aws-indexes/k2
